mlrun/api/launcher/server.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2023 MLRun Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from mlrun.launcher.abstract import AbstractLauncher
+
+
+class ServerSideLauncher(AbstractLauncher):
```

Comment:
```suggestion
class Launcher(AbstractLauncher):
```

It is enough here as enyway it "lives" under "api" which essentially means it is a server-side launcher.

Reply:
also, I think you can move it alltogether under `mlrun/api/launcher.py`